po/fr.po

```diff
@@ -2722,12 +2707,14 @@ msgstr "Indiquez les permissions du fichier mailbox"
 
 #, c-format
 msgid "%s warning: %s's uid %d is greater than SYS_UID_MAX %d\n"
-msgstr ""
+msgstr "%s attention: l'uid %d de %s est plus grand que SYS_UID_MAX %d\n"
 
 #, c-format
 msgid ""
 "%s warning: %s's uid %d outside of the UID_MIN %d and UID_MAX %d range.\n"
 msgstr ""
+"%s attention: l'uid %d de %s est en dehors de la plage UID_MIN %d et UID_MAX "
```

Comment:
There's a similar problem here.

Reply:
I'd suggest to use the following message:
`%s attention: l'uid de %s, %d, est en dehors de la plage UID_MIN %d et UID_MAX `